docs/sdk.rst

```diff
@@ -235,46 +235,26 @@ More details on the Globus Compute login manager prototcol are available `here.
 
   import globus_sdk
   from globus_sdk.scopes import AuthScopes
-  from globus_compute_sdk.sdk.login_manager import LoginManager
-  from globus_compute_sdk.sdk.web_client import WebClient
-  from globus_compute_sdk import Client
-
-  class LoginManager:
-    """
-    Implements the globus_compute_sdk.sdk.login_manager.protocol.LoginManagerProtocol class.
-    """
-
-    def __init__(self, authorizers: dict[str, globus_sdk.RefreshTokenAuthorizer]):
-        self.authorizers = authorizers
 
-    def get_auth_client(self) -> globus_sdk.AuthClient:
-        return globus_sdk.AuthClient(
-            authorizer=self.authorizers[AuthScopes.openid]
-        )
+  from globus_compute_sdk import Executor, Client
+  from globus_compute_sdk.sdk.login_manager import AuthorizerLoginManager
+  from globus_compute_sdk.sdk.login_manager.manager import ComputeScopeBuilder
 
-    def get_web_client(self, *, base_url: str) -> WebClient:
-        return WebClient(
-            base_url=base_url,
-            authorizer=self.authorizers[Client.FUNCX_SCOPE],
-        )
+  ComputeScopes = ComputeScopeBuilder()
 
-    def ensure_logged_in(self):
-        return True
+  # Create Authorizers from the Compute and Auth tokens
+  compute_auth = globus_sdk.AccessTokenAuthorizer(tokens[ComputeScopes.resource_server]['access_token'])
+  openid_auth = globus_sdk.AccessTokenAuthorizer(tokens[AuthScopes.openid]['access_token'])
```

Comment:
Where did `tokens` come from?  Is this provided by the user somehow?

Reply:
Good point. I originally decided getting tokens was out of scope, but I think it'd be useful to have more examples of things like this, so I'll add it.

Reply:
After our talk, I'm clearer now on the intended use-case for this class; what might helpful is an example or two of where the `tokens` object may have come, and perhaps a little surrounding context for where this might be utilized.  In particular, the "aha!" moment for me was when I finally understood this is intended for 3rd-party use of a token, so you might describe one or two example setups that motivate the need for/use of this class.  At which point, the `tokens` might be renamed to `token_from_client` or `user_auth_tokens_from_request`, or "something better than what I just blurted out."